sphinx_book_theme/layout.html

```diff
@@ -20,7 +20,25 @@
 <main class="col py-md-3 pl-md-4 bd-content overflow-auto" role="main">
     {% block docs_body %}
     {% include "topbar.html" %}
-    <div id="main-content" class="row">
+    <h1 class="onlyprint">{{ pagetitle }}</h1>
+    <!-- Table of contents -->
+    <div id="main-content" class="row onlyprint">
+        <div class="col-12 col-md-12 pl-md-5 pr-md-5">
+        <div>
+            {% set page_toc = generate_toc_html() %}
+            {%- if page_toc | length >= 1 %}
+            <div>
+                <h2> {{ translate(theme_toc_title) }} </h2>
+            </div>
+            <nav aria-label="Page">
```

Comment:
we probably don't need aria labels and such because this is being printed to PDF, so screen readers won't make sense, right?

Reply:
@choldgraf these lectures, I reckon, can also be just downloaded as PDF and read on a computer or tablet offline instead of printing it in particular? In which case, screen readers would be important. Not sure, if this HTML tag translates to pdf output as well though.

Reply:
I'm not sure either - but if we think it might help folks with a11y issues, let's just leave it in!